README.md

```diff
@@ -53,24 +54,56 @@ locals {
     ]
   ])
   
-  # This sample represents the nodes that will be used for insecure setup
+  # This sample represents the nodes that will be used for unsecured setup.
   list = flatten([
     for instance in module.ec2_instances: [
        "http://${instance.public_ip}:8080"
     ]
   ])
   
-  # This sample represents the nodes that will be used for secure setup
+  # This samples represents the nodes that will be used for secure setup.
   list = [for tag in local.nodes : "https://${tag}.omermichleviz.development.run"]
+    
 }
 ```
+#### This sample represents explicit assigning approach.
+
+```hcl
+locals {
+  
+  # Node tags
+  nodes = toset(["a", "b", "c"])
+  
+  # IP addresses for hosts to deploy RavenDB to
+  hosts = [
+         "3.95.238.149", 
+         "3.87.248.150", 
+         "3.95.220.189" 
+         ]
+  
+  # This sample represents the nodes that will be used for unsecured setup.
+  list = [
+         "http://3.95.238.149:8080", 
+         "http://3.87.248.150:8080", 
+         "http://3.95.220.189:8080"
+         ]
+  
+  # This samples represents the nodes that will be used for secure setup.
+  list = [
```

Comment:
can we make that `ravendbNodeUrls` ?

Reply:
We can. I think it's not needed because we have the table with all of the attribute explanations underneath.
Let's keep it as is for now ?

Reply:
I'm just saying change the local variable name, not the parameter field name